data_curation/augmentation.py

```diff
@@ -450,3 +450,61 @@ def generate_augmented_examples(
         augmented_examples.append(ex_code)
 
     return augmented_examples
+
+
+def build_augmented_loinc_files(
+    input_path: str,
+    config: schemas.LoincFileGenerationConfig,
+    num_lcn: int = 5,
+    num_sn: int = 5,
+    num_dn: int = 5,
+    output_path_base: str = "./data/training_files/augmented_loinc",
+) -> None:
+    """
+    Generates augmented LOINC data files for the long common names, short
+    common names, and display names based on the provided configurations.
+
+    :param path: The path to the base LOINC name file.
+    :param configs: Configuration dictionaries for long common names, short
+        common names, and display names.
+    :param num_lcn: The number of augmented long common names to generate.
+    :param num_sn: The number of augmented short common names to generate.
+    :param num_dn: The number of augmented display names to generate.
+    :param output_files_base: The base path for the output files.
+    :return: None
+    """
+
+    num_map = {"short_name": num_sn, "long_common_name": num_lcn, "display_name": num_dn}
+
+    # Read in data/loinc_lab_names.csv
```

Comment:
Super tiny nit: with the directory re-org, this is now `data/snoinc_extracts/loinc_lab_names_XXXX.csv`